tutorials/interactive/index.md

```diff
@@ -0,0 +1,781 @@
+again favorites the lake, drawing a solid star and increasing the count.
+
+The following shows both states.
+
+<center><img src="images/favorited-not-favorited.png" alt="the custom widget you'll create"></center><br>
+
+To accomplish this, you'll create a single custom widget that includes both the
+star and and the count. Tapping the star changes state for both widgets,
+so it works best for the same widget to manage both.
+
+You can find the starting code for the Lakes example here:
+
+* [`lib/main.dart`](https://raw.githubusercontent.com/flutter/website/master/_includes/_code/lakes/main.dart)
+* [`pubspec.yaml`](https://raw.githubusercontent.com/flutter/website/master/_includes/_code/lakes/pubspec.yaml)
+* [`lakes.jpg`](https://github.com/flutter/website/blob/master/_includes/_code/lakes/images/lake.jpg)
+
+<aside class="alert alert-success" markdown="1">
+<i class="fa fa-lightbulb-o"> </i> **Tip:**
+If you are new to Flutter, use these steps to prepare for this tutorial:
+
+* [Set up](https://flutter.io/setup/) your environment.
+* If you don't have a preferred IDE, we recommend
+  [IntelliJ community edition.](https://flutter.io/intellij-setup/) (It's free!)
+* Save the files in the list above to your computer.
+* [Create your first Flutter
+  app.](https://flutter.io/getting-started/#creating-your-first-flutter-app)
+* Replace the `lib/main.dart` and `pubspec.yaml` files in your project
+  with those provided.
+* Create an `images` directory in your project, and add `lakes.jpg`.
+* Once you have a connected and enabled device, or you've launched the
+  [iOS simulator](/setup/#set-up-the-ios-simulator) (part of the Flutter install),
+  you are good to go!
+</aside>
+
+### Contents
+
+* [Stateful and stateless widgets](#stateful-stateless)
+* [Creating a stateful widget](#creating-stateful-widget)
+  * [Step 1: Decide who manages the widget's state](#step-1)
+  * [Step 2: Subclass StatefulWidget](#step-2)
+  * [Step 3: Subclass the State object](#step-3)
+  * [Step 4: Plug the stateful widget into the widget tree](#step-4)
+  * [Problems?](#problems)
+* [Managing state](#managing-state)
+  * [The widget manages its own state](#self-managed)
+  * [The parent manages the widget's state](#parent-managed)
+  * [A mix-and-match approach](#mix-and-match)
+* [Other interactive widgets](#other-interactive-widgets)
+  * [Standard widgets](#standard-widgets)
+  * [Material widgets](#material-widgets)
+* [Resources](#resources)
+
+<a name="stateful-stateless"></a>
+## Stateful and stateless widgets
+
+<div class="panel" markdown="1">
+
+<b> <a id="whats-the-point" class="anchor" href="#whats-the-point" aria-hidden="true"><span class="octicon octicon-link"></span></a>What's the point?</b>
+
+* _Stateful_ means that a widget can change because of user interaction or
+  other factors, such a updating the UI using information from a service.
+* All widgets are either stateful or stateless.
+* A stateless widget sits there and looks pretty.
```

Comment:
hah This made me LOL.

Not sure it's quite accurate... an animation could move a stateless widget around the screen every frame.